src/components/Contacts/ContactsTable/ContactsTable.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box, Table, colors } from '@material-ui/core';
+import { ContactRow } from '../ContactRow';
+import { ContactRowFragment } from '../ContactRow/ContactRow.generated';
+
+interface Props {
+  contacts: ContactRowFragment[] | undefined;
+  loading: boolean;
+  error: Error;
+}
+
+const ContactsTable: React.FC<Props> = ({
+  contacts = [],
+  loading,
+  error,
+}: Props) => {
+  const renderLoading = () => (
+    <Box
+      data-testID="LoadingText"
+      height="100%"
+      alignItems="center"
+      justifyContent="center"
+      bgcolor={colors.green[600]}
+    >
+      Loading
+    </Box>
+  );
+
+  const renderEmpty = () => (
+    <Box data-testID="EmptyText" height="100%" bgcolor={colors.yellow[600]}>
+      No Data
+    </Box>
+  );
+
+  const renderError = () => (
+    <Box data-testID="ErrorText" bgcolor={colors.red[600]}>
+      Error: {error.toString()}
+    </Box>
+  );
+
+  return (
+    <Box height="100%">
```

Comment:
I'm having trouble getting the table to stretch and fill the rest of the window space.  `height="100%"` is ultimately not what I want -- it will make it the same height as the window, but since there is a header above this component, the bottom of this component extends beyond the view.  I've been trying to find the right combination of CSS and MaterialUI props to get the right effect, but no luck so far.  My React Native experience with flex boxes is not working here!

What I want:
- Scrolling disabled for the overall screen
- ContactsHeader should have fixed height
- ContactsTable should fill remaining available space
- ContactsTable should allow for scrolling inside itself, if the list of contacts is long

Reply:
I think we should make things sticky instead of nested scroll bars and trying to size things correctly to the screen. That way we just use the main body scroll which is less buggy.

Idk what Material has but something involving `position: sticky;` CSS or absolute positioning.

The AppBar supports it https://github.com/CruGlobal/mpdx-react/commit/c66ae9f7ef2abf9a5922943fd3958aa89689f086